documentation/airport-guides/lrop.md

```diff
@@ -0,0 +1,36 @@
+### About this facility
+Henri Coanda International Aiport (LROP / OTP), Otopeni, Bucharest, Romania, EU
+120,700 annual operations in 2016 ([BUCHAREST AIRPORTS COMPANY](http://www.bucharestairports.ro/ro/noutati/trafic-aerian-record-in-anul-2016))
+
+Located at about 7 miles north of Bucharest, the capital of Romania, it is the main international airport in the country. It has become the busiest airport since Baneasa Aurel Vlaicu Airport was converted in a general aviation airport and all low cost operations were moved to Otopeni. Traffic consists of medium aircrafts (mostly from A320 and B737 families) with seldom heavy arrivals.

```

Comment:
- `converted *into* a general aviation airport`
- `medium aircraft` (plural of aircraft is also aircraft)